src/app/containers/Ad/utilities/adSlotStyles.js

```diff
@@ -0,0 +1,91 @@
+import { css } from 'styled-components';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+// Ad Slot Widths for custom breakpoints
+const MPU_WIDTH_MIN = '19.75rem'; // 316px
+
+const LEADERBOARD_WIDTH_MIN = '21rem'; // 336px
+
+const LARGE_LEADERBOARD_WIDTH_MIN = '58.75rem'; // 940px
+
+// Ad Slot Heights
+/*
+  The height of the Ad Slot Label is taken into account for the total max height
+  of the Ad Slot. As of 13/08/2020, there are shared values between the MPU and Leaderboard heights,
+  however this may be subject to change in the future, hence the duplication.
+*/
+const AD_UNIT_MARGIN = 2.0625; // 25px
+
+const MPU_HEIGHTS = {
+  GROUP_1: `${3.125 + AD_UNIT_MARGIN}rem`,
+  // 50px + AD_UNIT_MARGIN = 75px
+  GROUP_2: `${3.125 + AD_UNIT_MARGIN}rem`,
+  // 50px + AD_UNIT_MARGIN = 75px
+  GROUP_3: `${15.625 + AD_UNIT_MARGIN}rem`,
+  // 250px + AD_UNIT_MARGIN = 275px
+  GROUP_4: `${15.625 + AD_UNIT_MARGIN}rem`,
```

Comment:
Could we replace `GROUP_4` here and below with `LARGE_LEADERBOARD` like we have done with the breakpoints? We're not using GEL group 4 here because of the ad sizes, so I don't want it to get confused :)